cmd/wallet-sdk-gomobile/README.md

```diff
@@ -26,6 +25,12 @@ This package contains the `gomobile`-compatible version of the SDK. It acts as a
 
 ## Generating the Bindings
 
+Change to the gomobile directory:
+```
+cd cmd/wallet-sdk-gomobile
```

Comment:
This README is already in `cmd/wallet-sdk-gomobile` (not at the root of the project, where we have a different README), so my assumption is that the user would already be in the `cmd/wallet-sdk-gomobile` folder.